pictures/desktop.ini

```diff
@@ -0,0 +1,33 @@
+[LocalizedFileNames]
```

Comment:
Is this a file your computer is automatically generating? If so, you might want to ignore it in git